tests/all/manifest.rs

```diff
@@ -157,6 +157,28 @@ fn it_creates_a_pkg_json_with_correct_files_on_node() {
     assert_eq!(actual_files, expected_files);
 }
 
+#[test]
+fn it_creates_a_pkg_json_in_out_dir() {
+    let out_dir = PathBuf::from("../ferris/is/awesome");
```

Comment:
For tests, we should probably use either

* a custom out directory that is still within the crate, eg "out" instead of "pkg", or
* a temporary directory with `tempfile::TempDir`.

What we don't want is to create random directories that might conflict with other tests and might not get cleaned up at the end of the tests.

Reply:
Well said. I will do a custom directory in the crate.